lib/sqlalchemy/dialects/mysql/mariadbconnector.py

```diff
@@ -118,6 +118,7 @@ class MySQLDialect_mariadbconnector(MySQLDialect):
     supports_native_decimal = True
     default_paramstyle = "qmark"
     execution_ctx_cls = MySQLExecutionContext_mariadbconnector
+    type_compiler_cls = MariaDBTypeCompiler
```

Comment:
I'm not 100% sure but I don't think this is the correct place for it.

@zzzeek Do you think we should add it dynamically when a dialect is used in mariadb using the loader function?